homeassistant/components/sensor/nut.py

```diff
@@ -0,0 +1,291 @@
+"""
+Provides a sensor to track various status aspects of a UPS.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.nut/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    CONF_HOST, CONF_PORT, CONF_NAME, CONF_USERNAME, CONF_PASSWORD,
+    TEMP_CELSIUS, CONF_RESOURCES, CONF_ALIAS, ATTR_STATE)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pynut2==2.1.2']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'NUT UPS'
+DEFAULT_HOST = 'localhost'
+DEFAULT_PORT = 3493
+
+KEY_STATUS = 'ups.status'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+
+SENSOR_TYPES = {
+    'ups.status': ['Status Data', '', 'mdi:information-outline'],
```

Comment:
Sorry, was talking about the entries for the `configuration.yaml` file. But I guess that it's simpler to use the entries from the third party module.